tdrs-backend/tdpservice/search_indexes/models/ssp.py

```diff
@@ -12,6 +12,11 @@ class SSP_M1(models.Model):
     Mapped to an elastic search index.
     """
 
+    class Meta:
+        """Meta class for the model."""
+
+        verbose_name = 'SSP M1'
```

Comment:
Are these necessary? I can't see a place where you were using this in the change-set. Is this an addition for future use/functionality coming down the pipes from the 508 work?

Reply:
This is to have a more readable name on admin page. If you login to admin, you will see that change. As an example 'SSP M1' used to show: sspm1 which is not nice.